src/Symfony/Component/Validator/Validator/RecursiveContextualValidator.php

```diff
@@ -305,6 +305,10 @@ protected function normalizeGroups($groups)
     private function validateObject($object, string $propertyPath, array $groups, int $traversalStrategy, ExecutionContextInterface $context)
     {
         try {
+            if (!\is_object($object)) {
```

Comment:
This execution path is missing a test case.

Test case would probably:

 1. register an autoloader
 2. verify if said autoloader is triggered with malformed input
 3. unregister autoloader

Reply:
Yeah sorry but that goes beyond the time I'm willing to invest in this :D If someone wants to add a test please be my guest.

Reply:
That's understandable, it's not like these kind of issues appeared/regressed because of a lack of testing or such :trollface:

Reply:
`@param object $object`, so this seems suspicious :)

Reply:
ohh too quick:

>  it's explicitly passing anything in there to get the proper exception

but still :)

Reply:
Yes and no.. if you do expect an object and you have no type hint, validing that you did indeed get an object and throwing otherwise doesn't seem that unreasonable.

Reply:
but as we explicitly use that private method with non-objects to get the error message it generates, I would change the phpdoc to `mixed` instead (we don't want that exception to become a TypeError due to adding a native `object` typehint)